examples/tabs/index.html

```diff
@@ -1,582 +1,402 @@
+    <p>
+        Using option <code>orientation</code> we can set how the tabs and body should be positioned.
+    </p>
+    <script id="script-tabWidget-option-orientation">
+      window.addEventListener('DOMContentLoaded', function createTabsWithOnClose () {
+        const demo = document.querySelector('#demo-tabWidget-option-orientation')
+        const item1 = UI.rdf.namedNode(location.origin + '#item3A')
+        const item2 = UI.rdf.namedNode(location.origin + '#item3B')
+        demo.appendChild(UI.tabs.tabWidget({
+          items: [item1, item2],
+          orientation: '1', // tabs shown to the left of body
+          renderMain: (bodyMain, subject) => {
+            bodyMain.innerText = `Content of ${subject}`
+          }
+        }))
+        demo.appendChild(UI.tabs.tabWidget({
+          items: [item1, item2],
+          orientation: '2', // tabs shown below body
+          renderMain: (bodyMain, subject) => {
+            bodyMain.innerText = `Content of ${subject}`
+          }
+        }))
+        demo.appendChild(UI.tabs.tabWidget({
+          items: [item1, item2],
+          orientation: '3', // tabs shown to the right of body
+          renderMain: (bodyMain, subject) => {
+            bodyMain.innerText = `Content of ${subject}`
+          }
+        }))
+      })
+    </script>
+    <pre id="viewSource-tabWidget-option-orientation"></pre>
+    <div id="demo-tabWidget-option-orientation"></div>
+</article>
+
+<article id="tabWidget-option-renderTab">
+    <h2>
+        <a href="#tabWidget-option-renderTab"><code>tabWidget</code>: Option <code>renderTab</code></a>
+    </h2>
+    <p>
+        <code>renderTab</code> allows us to override the default behavior for how tabs are rendered, e.g.
+        which text should be shown for each tab.
+    </p>
+    <script id="script-tabWidget-option-renderTab">
+      window.addEventListener('DOMContentLoaded', function createTabsWithOnClose () {
+        const demo = document.querySelector('#demo-tabWidget-option-renderTab')
+        const item1 = UI.rdf.namedNode(location.origin + '#item3A')
+        const item2 = UI.rdf.namedNode(location.origin + '#item3B')
+        demo.appendChild(UI.tabs.tabWidget({
+          items: [item1, item2],
+          renderMain: (bodyMain, subject) => {
+            bodyMain.innerText = `Content of ${subject}`
+          },
+          renderTab: (tabDiv, subject) => {
+            tabDiv.innerText = subject.uri
+          }
+        }))
+      })
+    </script>
+    <pre id="viewSource-tabWidget-option-renderTab"></pre>
+    <div id="demo-tabWidget-option-renderTab"></div>
+</article>
+
+<article id="tabWidget-option-renderTabSettings">
+    <h2>
+        <a href="#tabWidget-option-renderTabSettings"><code>tabWidget</code>: Option <code>renderTabSettings</code></a>
+    </h2>
+    <p>
+        <code>renderTabSettings</code> functions much like <code>renderMain</code>, expect that it is triggered by
+        holding ALT key and clicking on a tab.
```

Comment:
(or 'option' on a Mac)

Reply:
and wow, never knew this functionality existed! :)

Reply:
Heh, yeah, there are some magic in the existing solution.